server/app.js

```diff
@@ -10,13 +10,24 @@ const pingRouter = require("./routes/ping");
 const { json, urlencoded } = express;
 
 var app = express();
+var mongoose = require("mongoose"); //For database connection
+
+// Connecting to database. fail if not able to connect
+mongoose.connect("mongodb://localhost:27017", {useNewUrlParser:true});
+var db = mongoose.connection;
```

Comment:
I know we used var above in our code, but let's consistently use either let or const. We want to favor using const over let (especially if we never need to reuse it) - so in these cases we want to use const

Reply:
Understood. Updated var to const.